test/kino/process_test.exs

```diff
@@ -24,6 +24,24 @@ defmodule Kino.ProcessTest do
       assert content =~ "0(supervisor_parent):::root ---> 2(#{inspect(agent)}):::worker"
     end
 
+    test "shows supervision tree with children alongside non-started children" do
+      {:ok, pid} =
+        Supervisor.start_link(
+          [
+            {Agent, fn -> :ok end},
+            %{id: :not_started, start: {__MODULE__, :start_ignore, []}}
```

Comment:
Would this work?

```suggestion
            %{id: :not_started, start: {Function, :identity, [:ignore]}}
```

Reply:
Looks like it works great! You kinda know this language pretty well, huh?